src/time/Waveform.hpp

```diff
@@ -114,6 +103,21 @@ class Waveform {
    * @return Order that may be used.
    */
   int computeUsedOrder(int requestedOrder, int numberOfAvailableSamples);
+
+  /**
+   * @brief Returns point the closest time stored in _timeStepsStorage that is after normalizedDt
+   *
+   * @param normalizedDt point in time
+   * @return double point in time after normalizedDt in _timeStepsStorage
```

Comment:
```suggestion
   * @return double closest point in time
```
No need to duplicate information from the docstring above.